### PR TITLE
[CRM457-1543] Allow optional partial comment|

### PR DESCRIPTION
**Pull request by dwhenry**
## Description of change
Allow partial comment to be optional - update comment field label as per design

 [Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1543)

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -7,7 +7,7 @@
 DATABASE_URL=postgresql://postgres@localhost/laa_assess_non_standard_magistrate_fee_dev
 
 # Local datastore url - default is http://localhost:8000
-APP_STORE_URL=http://localhost:8000
+APP_STORE_URL=https://localhost:3002
```
Comment (jsugarman) [inline .env.development:10]: can we remove this for the same reasons as per similar PR. your local preferences can be put in `env.development.local` if you need to or want to try stuff out

File: .env.development [hunk 2]
```diff
@@ -40,11 +40,11 @@ ENABLE_SYNC_TRIGGER_ENDPOINT=true
...
-CASEWORKER_CLIENT_ID=not-a-real-key
-CASEWORKER_CLIENT_SECRET=not-a-real-key
+APP_STORE_CLIENT_ID=
+APP_STORE_TENANT_ID=
+CASEWORKER_CLIENT_ID=
+CASEWORKER_CLIENT_SECRET=
 
 # ingress and protocol for webhook subscription to app store - must match local apps's rails server
-HOSTS=localhost:3002
+HOSTS=https://localhost:3001
```
Comment (jsugarman) [inline .env.development:49]: [ditto](https://github.com/ministryofjustice/laa-assess-crime-forms/pull/498/files#r1624046397)

**Review by jsugarman — COMMENTED** [review]
is this ready to review, LGTM other than the config changes, as per similar previous PR

**Comment by patrick-laa** [comment]
I'm closing this because:
It's been open for a month
The [ticket](https://dsdmoj.atlassian.net/browse/CRM457-1543) associated with this PR is marked as done
The ticket is about changes to the PA journey, and this modifies the NSM journey

My inference from all of the above is that this PR is surplus to requirements.

**Pull request closed without merging**